package.json

```diff
@@ -23,6 +23,7 @@
   },
   "homepage": "https://github.com/digitalbazaar/bedrock-authn-token-http",
   "dependencies": {
+    "bedrock-validation": "^5.5.0",
```

Comment:
Bedrock libs should always be peer dependencies. Adding them as regular dependencies can result in duplicate libs being installed and multiple libs being loaded into memory -- breaking applications. Bedrock apps expect there to be exactly 1 version of any bedrock lib as they are stateful plugins into a bedrock framework that communicates across shared modules (bedrock events, etc.) -- as opposed to just regular code modules that provide isolated functions.

```suggestion
```

Reply:
Updated